internal/app/donor_store_test.go

```diff
@@ -832,3 +837,52 @@ func TestDonorStoreDeleteWhenSessionMissing(t *testing.T) {
 		})
 	}
 }
+
+func TestEventSchema(t *testing.T) {
+	testcases := map[string]any{
+		"application-updated": applicationUpdatedEvent{
+			UID:       "M-0000-0000-0000",
+			Type:      "hw",
+			CreatedAt: time.Now(),
+			Donor: applicationUpdatedEventDonor{
+				FirstNames:  "syz",
+				LastName:    "syz",
+				DateOfBirth: date.New("2000", "01", "01"),
+				Address: place.Address{
+					Line1:      "line1",
+					Line2:      "line2",
+					Line3:      "line3",
+					TownOrCity: "townOrCity",
```

Comment:
I wanted to be sure that the `json:` tag for this doesn't get changed without realising it will affect the event